configs/katello_36.yaml

```diff
@@ -0,0 +1,202 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.4.5:
+    :redmine_release_id: 258
+  :3.4.6:
+    :redmine_release_id: 295
+  :3.5.0:
+    :redmine_release_id: 250
+:releases:
+  :3.6.0:
+    :redmine_release_id: 284
+:code_name: Imperial IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.11.4
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/develop
+    :repo: https://github.com/Katello/katello-packaging.git
+    :version_branch: true
+  :katello-agent:
+    :branch: 3.1.0
+    :repo: https://github.com/Katello/katello-agent.git
+  :bastion:
+    :branch: 6.1.0
+    :repo: https://github.com/Katello/bastion
+  :puppet-pulp:
+    :branch: 5.1.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-foreman_proxy_content:
+    :branch: 6.0.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 7.0.0
+    :repo: https://github.com/Katello/puppet-katello
+  :puppet-candlepin:
+    :branch: 4.0.0
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 3.0.1
+    :repo: https://github.com/Katello/puppet-certs
+
+:ignores:
+  - 20792
+:gpg_key: '3DA3BBF0'
```

Comment:
This will change when we generate a new key for GA. Maybe we should just set it to an empty string.